components/Common/Badge/index.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ComponentProps, FC, PropsWithChildren } from 'react';
```

Comment:
Keep an eye on the order of imports.

Usually type imports are the last ones; then style imports happen after all imports on a new line 👀